tracer/build/artifacts/dd-dotnet.sh

```diff
@@ -47,10 +75,13 @@ fi
 
 # Check if DD_DOTNET_PATH is set and the file exists
 if [ -z "$DD_DOTNET_PATH" ]; then
+    # Should never happen
```

Comment:
I think we said about detecting macos too?

Reply:
At the beginning:

```
# Check the OS
OS_NAME="$(uname -s)"
if [ "$OS_NAME" != "Linux" ]; then
    echo "This script is intended for Linux systems only. Current system is $OS_NAME."
    exit 1
fi
```

Reply:
_shakes fist at GitHub collapsing the file_ 
![image](https://github.com/DataDog/dd-trace-dotnet/assets/18755388/3a91e1cb-9256-4cfa-ae6d-a9dc7a2079ee)